### PR TITLE
feature/MBA-7_sign_in_screen

### PR DESCRIPTION
**Pull request by stepan103ua**
image

### CODE REVIEW

**Review by Xaiodo — APPROVED** [review]
LGTM

File: src/screens/HomeScreen/SignInScreen/AuthCreateAccount/authCreateAccount.styles.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { colors } from '../../../../constants/colors';
+import { responsiveHeight } from '../../../../constants/dimensions';
+
+const styles = StyleSheet.create({
+  container: {
+    height: responsiveHeight(14),
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+  },
+  touchContainer: {},
```
Comment (milkanuch) [inline src/screens/HomeScreen/SignInScreen/AuthCreateAccount/authCreateAccount.styles.ts]: empty styles?

File: src/screens/HomeScreen/SignInScreen/AuthHeader/AuthHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { View, Text } from 'react-native';
+import { FC } from 'react';
+
+import styles from './authHeader.styles';
+
+const AuthHeader: FC = () => {
```
Comment (milkanuch) [inline src/screens/HomeScreen/SignInScreen/AuthHeader/AuthHeader.tsx]: :FC is not needed here

**Review by milkanuch — APPROVED** [review]
LGTM

**Pull request merged**